examples/dt/nova/nova02beta/edpm/nodeset2/values.yaml

```diff
@@ -0,0 +1,162 @@
+# yamllint disable rule:line-length
+# local-config: referenced, but not emitted by kustomize
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edpm-nodeset-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  root_password: cmVkaGF0Cg==
+  preProvisioned: false
+  baremetalSetTemplate:
+    ctlplaneInterface: eno2  # CHANGEME
+    cloudUserName: cloud-admin
+    provisioningInterface: enp1s0  # CHANGEME
+    bmhLabelSelector:
+      app: openstack  # CHANGEME
+    passwordSecret:
+      name: baremetalset-password-secret
+      namespace: openstack
+  ssh_keys:
+    # Authorized keys that will have access to the dataplane computes via SSH
+    authorized: CHANGEME
+    # The private key that will have access to the dataplane computes via SSH
+    private: CHANGEME2
+    # The public key that will have access to the dataplane computes via SSH
+    public: CHANGEME3
+  nodeset:
+    ansible:
+      ansibleUser: cloud-admin
+      ansiblePort: 22
+      ansibleVars:
+        # CHANGEME -- see https://access.redhat.com/solutions/253273
+        # edpm_bootstrap_command: |
+        #       subscription-manager register --username <subscription_manager_username> --password <subscription_manager_password>
+        #       podman login -u <registry_username> -p <registry_password> registry.redhat.io
+        timesync_ntp_servers:
+          - hostname: pool.ntp.org
+        # CPU pinning settings
+        edpm_kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=16 intel_iommu=on iommu=pt isolcpus=4-23,28-47"
```

Comment:
isolcpus shoudl not be used on non realtime systems or on non nfv systems
we expicltly consdier that to be a bad practics and technislly unsupproted by the comptue team.

please do not use this in any nova DT or va.

edpm_tuned_isolated_cores is the recommend way to isolate the cores on a non realtime system.

Reply:
also keep in mind that neither edpm_tuned_isolated_cores nor isolcpus is required to use cpu pinning. edpm_tuned_isolated_cores is recommend but still optional.

customers repetitively misuse isolcpus which is why we intentionally removed it form the comptue docs and why we shoudl avoid using it in our VAs 

this was especially true of telcos who read the nfv docs but did not understand how ti worked and they wound of ten fil bug reports for poor performance of there floating vm because they incorrectly included core from cpu_share_set in the iscolcpu range.

this was espically common for osp 16.2 users im not sure if the decress in supprot request in 17.1 is becasue of better docs or if they just learned there leason on 16.